src/util/fetchUtils.js

```diff
@@ -87,8 +87,8 @@ export const getFetchArgs = (apiUrl, queries, meta) => {
 	const isFormData = queries[0].params instanceof FormData;
 	const isDelete = method === 'DELETE';
 
-	const fetchUrl = new URL(apiUrl);
-	fetchUrl.searchParams.append('queries', rison.encode_array(queries));
+	const searchParams = new URLSearchParams();
```

Comment:
`fetchUtils` only runs in the browser - the app server exclusively uses the `request` lib for server-to-server requests, not `fetch` - `window` is safe to use here. Probably should move this comment up to the top of the module, but browser-only usage is declared [here](https://github.com/meetup/meetup-web-platform/blob/6fd9c7ba88dd3c1a2e7c85094f1413f7c47cebfa/src/util/fetchUtils.js#L154-L155)